File: python/algorithms/sorts/insertion/insert.py
def insertion_sort(seq):
    for i in range(1, len(seq)):
        key = seq[i]
        j = i-1
        while j >= 0 and seq[j] > key:
            seq[j+1] = seq[j]
            j -= 1
        seq[j+1] = key

File: c++/algorithms/sorts/insertion/insert.cpp
#include "../sorts.h"

void insertion_sort(int* arr, int size)
{
	for (int i = 1; i < size; ++i) {
		int key = arr[i], j;
		for (j = i-1; j >= 0 && arr[j] > key; --j) {
			arr[j+1] = arr[j];
		}
		arr[j+1] = key;
	}
}

